src/index.html

```diff
@@ -9,6 +9,33 @@
       gtag('js', new Date());
       gtag('config', '{{ WALLET_GA_MEASUREMENT_ID || "" }}');
     </script>
+    <script type="text/javascript">
```

Comment:
is there anything done to prevent tracking on development / staging environment?

Reply:
Should I use the same template tag approach as you did with GA? Wasn't quite sure how we're handling env variables in production

Reply:
same approach can work, but make sure it does disable tracking (vs crashing) when used in prod